SU2_DEF/include/drivers/CDiscAdjDeformationDriver.hpp

```diff
@@ -0,0 +1,155 @@
+/*!
+* \file CDiscAdjDeformationDriver.cpp
+* \brief Main subroutines for driving the projection of sensitivities.
+ * \author T. Economon, H. Kline, R. Sanchez
+ * \version 7.1.1 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2021, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define ENABLE_MAPS
+#include "../../../Common/include/CConfig.hpp"
+#undef ENABLE_MAPS
```

Comment:
we should only do this ENABLE_MAPS in .cpp files to keep it from spreading.
(the const static maps we have in option_structure become a compilation problem if used everywhere)

Reply:
It looks like some of my old comments are still applicable. Given that this was in the backburner for a long time maybe you can take a look at those comments first.